src/Import/Remote/Composer.php

```diff
@@ -9,41 +9,43 @@
 use Psr\Log\NullLogger;
 use Symfony\Component\Console\Helper\ProgressIndicator;
 use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\Output;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Filesystem\Filesystem;
 
 /** @internal */
 class Composer
 {
-    public const VENDOR_DIR = '/.castor/vendor/';
+    public const VENDOR_DIR = '.castor/vendor';
 
     public function __construct(
         private readonly Filesystem $filesystem,
+        private readonly InputInterface $input,
         private readonly OutputInterface $output,
         private readonly LoggerInterface $logger = new NullLogger(),
     ) {
     }
 
-    public function install(string $entrypointDirectory, bool $update = false, bool $displayProgress = true): void
+    public function install(string $entrypointDirectory): void
     {
-        $vendorDirectory = $entrypointDirectory . self::VENDOR_DIR;
+        $update = true !== $this->input->getParameterOption('--update-remotes', true);
+        $displayProgress = 'list' !== $this->input->getFirstArgument();
```

Comment:
Why? It helps to understand why the list command could take some on the first run instead of outputing nothing.

If this to avoid to break the generate-test.sh , may be should we disable it only when the format option is passed?

Reply:
I just move some code from [Kernel](https://github.com/jolicode/castor/blob/48b138a67b6b82cc843719a7a7b017092e4b69fc/src/Kernel.php#L91-L92)

Reply:
It was already there before, let's keep it for the moment and remove it in another PR if we want to change this behavior